packages/angular_devkit/build_angular/src/builders/karma/application_builder.ts

```diff
@@ -356,25 +376,16 @@ async function initializeApplication(
     { pattern: `${outputPath}/polyfills.js`, type: 'module', watched: false },
     // Serve global setup script.
     { pattern: `${outputPath}/${mainName}.js`, type: 'module', watched: false },
-    // Serve all source maps.
-    { pattern: `${outputPath}/*.map`, included: false, watched: false },
```

Comment:
Question: Are we intentionally removing sourcemaps here?

Reply:
Originally: Yes. I assumed that serving static files using the Karma middleware would "just work" now. But Karma is certainly... creative and the one thing I can say with certainty is that it doesn't _just work_. So, given how annoying it is to test source maps in particular, let me back out these changes.